v19.1/managed-authentication.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Authentication
+summary: Learn about the authentication features for Managed CockroachDB clusters.
+toc: true
+build_for: [managed]
+---
+
+Managed CockroachDB uses TLS 1.2 digital certificates for inter-node authentication, [SSL modes](https://www.postgresql.org/docs/11/libpq-ssl.html) for node identity verification, and password authentication for client identity verification.
+
+## Node identity verification
+
+Managed CockroachDB uses the `verify-full` [SSL mode](https://www.postgresql.org/docs/11/libpq-ssl.html) by default to verify a node’s identity. This mode encrypts the data in-flight as well as verifies the identity of the CockroachDB node, thus ensuring a secure connection to your cluster. Using this mode prevents MITM (Man in the Middle) attacks, impersonation attacks, and eavesdropping.
```

Comment:
This is more about recommending `verify-full` than using.
The only mode we cannot support is `disable` as that requires plaintext. All other modes with work just fine with our clusters.

Reply:
This para talks about the SSL mode we use by default (`verify-full`) and what the user needs to do about it. I do include info about using the other supported mode (`require`) in the next para.

Also, I tried the `disable`, `allow`, and `prefer` modes and got the `unsupported sslmode` error.

Reply:
I know, I was talking more about the phrasing. We don't use and can't enforce which mode they use other than `disable`.
The errors come from libpq which does not support `allow` and `prefer` as it can't switch dynamically between TLS and plaintext. `disable` just won't work with our clusters.
If you try with `psql`, you should be able to connect with everything other than `disable`.

Reply:
Ahh..okay. Does this sound better: "Use the `verify-full` SSL mode to verify a node's identity"